### PR TITLE
Add some implementations on CloudCore

### PR DESCRIPTION
**Pull request by ls889**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test

/kind feature

**What this PR does / why we need it**:
Add a specific implementation on CloudCore.

**Which issue(s) this PR fixes**:

Fixes #

Ref: #1443

### CODE REVIEW

File: cloud/pkg/cloudhub/cloudhub.go [hunk 1]
```diff
@@ -64,7 +64,9 @@ func (a *cloudHub) Start() {
 	// start dispatch message from the cloud to edge node
 	go messageq.DispatchMessage()
 
-	httpserver.SignCerts()
+	httpserver.PrepareForAllCert()
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/cloudhub.go]: `PrepareForAllCerts`

File: cloud/pkg/cloudhub/servers/httpserver/server.go [hunk 2]
```diff
@@ -17,27 +17,40 @@ limitations under the License.
 package httpserver
 
 import (
+	"crypto"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
 	"fmt"
+	hubconfig "github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/config"
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/server.go]: move this line below L33

File: cloud/pkg/cloudhub/servers/httpserver/signcerts.go [hunk 3]
```diff
@@ -17,17 +17,37 @@ limitations under the License.
 package httpserver
 
 import (
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
 	"fmt"
+	hubconfig "github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/config"
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/signcerts.go]: ditto

File: cloud/pkg/cloudhub/servers/httpserver/updateconfig.go [hunk 4]
```diff
@@ -0,0 +1,22 @@
+package httpserver
+
+import (
+	"bytes"
+	hubconfig "github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/config"
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/servers/httpserver/updateconfig.go:6]: empty line above

File: cloud/pkg/cloudhub/config/config.go [hunk 5]
```diff
@@ -20,16 +20,25 @@ type Configure struct {
 	v1alpha1.CloudHub
 	KubeAPIConfig *v1alpha1.KubeAPIConfig
 	Ca            []byte
+	CaKey         []byte
 	Cert          []byte
 	Key           []byte
 }
 
+const (
+	rootCaFile= "/etc/kubeedge/ca/rootCA.key"
```
Comment (fisherxu) [inline cloud/pkg/cloudhub/config/config.go]: `rootCaKeyFile` ?

Comment (ls889) [reply cloud/pkg/cloudhub/config/config.go]: yes, I will write it in cloudhubconfig like ca and servercerts and delete the delete these two lines

**Pull request merged**